module/models/actor-base.mjs

```diff
@@ -111,6 +111,9 @@ export default class CrucibleBaseActor extends foundry.abstract.TypeDataModel {
     // Status
     schema.status = new fields.ObjectField({nullable: true, initial: null});
     schema.favorites = new fields.SetField(new fields.StringField({blank: false}));
+
+    // Languages
+    schema.languages = new fields.SetField(new fields.StringField({blank: false}));
```

Comment:
I think I'd like to put this under `system.details.languages` instead of just `system.languages`

Reply:
Yeah, both